central/reports/metadata/datastore/store/store.go

```diff
@@ -0,0 +1,23 @@
+package store
```

Comment:
Won't be able to mock the store if the one in postgres/store.go is used because it doesn't have the `//go:generate mockgen-wrapper` annotation. But I don't think that should be needed because we mock the datastore for most testing.

Reply:
Alright, I believe it will be addressed soon through the code removal and streamlining effort.